File: LeetCode/java/2729_Check_if_The_Number_is_Fascinating.java
class Solution {
    public boolean isFascinating(int n) {
        int nDouble = n * 2;
        int nTriple = n * 3;

        String s = String.valueOf(n) + String.valueOf(nDouble) + String.valueOf(nTriple);

        int[] list = new int[10];

        for (int i = 0; i < s.length(); i++) {
            list[s.charAt(i) - '0']++;
        }

        for (int i = 1; i <= 9; i++) {
            if (list[i] != 1) {
                return false;
            }
        }

        return true;
    }
}

File: LeetCode/cpp/2729_Check_if_The_Number_is_Fascinating.cpp
class Solution {
public:
	bool isFascinating(int n) {
		int n_double = n * 2;
		int n_triple = n * 3;

		string s = to_string(n) + to_string(n_double) + to_string(n_triple);

		vector<int> v(10, 0);

		for (int i = 0; i < s.length(); i++) {
			v[s[i] - '0']++;
		}

		for (int i = 1; i <= 9; i++) {
			if (v[i] != 1) {
				return false;
			}
		}

		return true;
	}
};